types/three/src/materials/MeshPhysicalMaterial.d.ts

```diff
@@ -16,17 +16,29 @@ export interface MeshPhysicalMaterialParameters extends MeshStandardMaterialPara
 
     sheen?: number | undefined;
     sheenColor?: Color | undefined;
+    sheneColorMap?: Texture | null | undefined;
     sheenRoughness?: number | undefined;
+    sheneRoughnessMap?: Texture | null | undefined;
 
     transmission?: number | undefined;
     transmissionMap?: Texture | null | undefined;
+
+    thickness?: number | undefined;
+    thicknessMap?: Texture | null | undefined;
+
     attenuationDistance?: number | undefined;
     attenuationColor?: Color | undefined;
 
     specularIntensity?: number | undefined;
     specularColor?: Color | undefined;
     specularIntensityMap?: Texture | null | undefined;
     specularColorMap?: Texture | null | undefined;
+
+    iridescenceMap?: Texture | null;
+    iridescenceIOR?: number;
+    iridescence?: number;
+    iridescenceThicknessRange?: number[];
+    iridescenceThicknessMap?: Texture | null;
```

Comment:
these properties should have `| undefined` as other properties do imo.

Reply:
sorry, my mistake.